cookbooks/aws-parallelcluster-platform/recipes/config/log_rotation.rb

```diff
@@ -24,7 +24,10 @@
 when 'ComputeFleet'
   include_recipe 'aws-parallelcluster-platform::log_rotation_compute_fleet'
 
+when 'LoginNode'
+  Chef::Log.info("Log rotation not yet implemented for LoginNodes")
```

Comment:
Will this be implemented in MVP?
If not, would this miss introduce any risk for the node?

Reply:
It should already be implemented. Jacopo is working on this.
I just added this fix to make the code work since he didn't publish the PR yet.